packages/Tabs/Tabs.md

```diff
@@ -0,0 +1,21 @@
+### Usage criteria
+
+Use `Tabs` according to props data
+Note that the `copy` prop must be provided at all times for the correct accessible text.
+
+```jsx
+<Tabs copy="en" tabs={['Themepacks', 'Premium', 'Essentials']}>
```

Comment:
One more change please @sudhirraivns0103 - can you make the heading for each panel an attribute of the panel rather than at the parent level?

So `<Tabs copy="en"><Tabs.Panel heading="Themepacks">Content</Tabs.Panel>...` 

like that.

Reply:
Hi @mike-bunce,
I have made the requested changes in the tabs component